s3gw/go.mod

```diff
@@ -1,40 +1,39 @@
 module github.com/cybozu/neco-containers/s3gw
 
-go 1.20
+go 1.22.1
```

Comment:
Patch versions should not be set.

Reply:
After installing Go 1.21, we never have to manually download and install a Go toolchain again. 
The go command can do it for you. but, we must specify such as 1.21.0.

Please refer this link for detail 
https://go.dev/blog/toolchain